src/webgpu/api/validation/capability_checks/limits/maxColorAttachmentBytesPerSample.spec.ts

```diff
@@ -0,0 +1,188 @@
+import { range } from '../../../../../common/util/util.js';
+import { roundDown } from '../../../../util/math.js';
+
+import { kLimitBaseParams, kLimitModes, makeLimitTestGroup } from './limit_utils.js';
+
+function getPipelineDescriptor(
+  device: GPUDevice,
+  actualLimit: number,
+  testValue: number
+): GPURenderPipelineDescriptor | undefined {
+  const numRGBA32F = actualLimit / 16;
+  const numR8Unorm = testValue - actualLimit;
+  const maxColorAttachments = device.limits.maxColorAttachments;
+  if (numRGBA32F + numR8Unorm > maxColorAttachments) {
+    return;
+  }
+
+  const rgba32fTargets = new Array(numRGBA32F).fill({
+    format: 'rgba32float',
+    writeMask: 0,
+  }) as GPUColorTargetState[];
+
+  const r8UnormTargets = new Array(numR8Unorm).fill({
+    format: 'rgba8unorm',
```

Comment:
```suggestion
    format: 'r8unorm',
```
? since I think you want a cost of 1, not 4

Reply:
doh! thanks. Fixed